connector/src/main/scala/com/microsoft/kusto/spark/common/KustoOptions.scala

```diff
@@ -4,18 +4,23 @@ import java.util.Locale
 
 trait KustoOptions {
   private val kustoOptionNames = collection.mutable.Set[String]()
-
   protected def newOption(name: String): String = {
     kustoOptionNames += name.toLowerCase(Locale.ROOT)
     name
   }
 
-  // KeyVault options. Relevant only if credentials need to be retrieved from Key Vault
-  val KEY_VAULT_URI = "keyVaultUri"
-  val KEY_VAULT_APP_ID = "keyVaultAppId"
-  val KEY_VAULT_APP_KEY = "keyVaultAppKey"
+  /** Required options */
+  // Target/source Kusto cluster for writing/reading the data.
+  val KUSTO_CLUSTER: String = newOption("kustoCluster")
 
+  // Target/source Kusto database for writing/reading the data. See KustoSink.md/KustoSource.md for
+  // required permissions
+  val KUSTO_DATABASE: String = newOption("kustoDatabase")
 
+  /**
+   * Authentication parameters - If non provided - device code would show on console. If console is not available
+   * use DeviceAuthentication class-https://github.com/Azure/azure-kusto-spark/blob/56667c54dfd43455113a9b48725b236a8d92ccd4/samples/src/main/python/pyKusto.py#L147
```

Comment:
Better to provide link to master branch (https://github.com/Azure/azure-kusto-spark/blob/master/samples/src/main/python/pyKusto.py)

Reply:
I really prefer not to - this way i can provide link to specific line

Reply:
You can do the same thing with the master branch:
https://github.com/Azure/azure-kusto-spark/blob/master/samples/src/main/python/pyKusto.py#L147

Reply:
the difference is - this doesnt gurantee the line will stay on the same line

Reply:
Good point!